src/server/server.js

```diff
@@ -0,0 +1,35 @@
+const  express = require('express');
+
+var middlewares = require("./middlewares");
+var controllers = require('./controllers');
+var api_rest = require('./controllers/api');
+var bodyParser = require('body-parser')
+var mongoose = require('mongoose');
+let port = 3000;
```

Comment:
No, I made a mistake in this case, all these vars could be declared as const because they won't be reassigned.